### DISCUSSION TITLE
有关阶段一完成指标的疑问

### DISCUSSION THREAD

**Discussion by ghost** [liusy58/NJU_AdvancedProgramming_SP22#181] [Q&A] [answered]
请问，阶段一如果选择完成ppt，ppt是否参与最终的评分？（是只需要完成就不扣分，还是说ppt是10分的组成部分）
如果参与，ppt要做到怎样才能算完成要求？有没有明确的量化的标准？

**Comment by liusy58** [comment, marked as answer]
PPT不算计分，这是为了给大家一个缓冲时间，还是按照提交的code情况进行计分

**Reply by ghost** [reply]
thanks
